tomcat/src/main/java/com/linecorp/armeria/server/tomcat/UnmanagedTomcatService.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2018 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.server.tomcat;
+
+import java.util.Optional;
+
+import javax.annotation.Nullable;
+
+import org.apache.catalina.connector.Connector;
+import org.apache.catalina.startup.Tomcat;
+
+public final class UnmanagedTomcatService extends TomcatService {
+    @Nullable
+    private Tomcat tomcat;
+    @Nullable
+    private String hostName;
+    @Nullable
+    private Connector connector;
+
+    UnmanagedTomcatService(Tomcat tomcat) {
+        this.tomcat = tomcat;
```

Comment:
I think this can be `this(tomcat.getEngine().getDefaultHost(), tomcat.getConnector())` and the other methods are just getters.

Reply:
The reason as mention as mentioned in the **Summary**, is because during Spring startup, it removes the connectors from the `Tomcat`, and after startup ends re-associates them.